test/connectors/BackLinkCacheConnectorSpec.scala

```diff
@@ -69,9 +75,8 @@ class BackLinkCacheConnectorSpec extends PlaySpec with GuiceOneServerPerSuite wi
         when(mockSessionCache.cache[ReviewDetails](any(), any())(any(), any(), any()))
           .thenReturn(Future.successful(returnedCacheMap))
 
-        when(mockHttpClient.PUT[BackLinkModel, CacheMap]
-          (any(), any(), any())(any(), any(), any(), any()))
-          .thenReturn(Future.successful(CacheMap("test", Map("BC_Back_Link:testPageId" -> Json.toJson(BackLinkModel(Some("testBackLink")))))))
+        when(mockHttpClient.put(any())(any)).thenReturn(requestBuilder)
```

Comment:
not needed, already tsaking place in ConnectorTest class

Reply:
Resolved.